site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -361,9 +361,7 @@ Run the following command to uninstall the Milvus Operator.
 $ helm -n milvus-operator uninstall milvus-operator
 ```
 
-Run the following command to uninstall the Milvus Operator.
-
-### Uninstall the Milvus Operator by kubectl command
```

Comment:
Remove the article "the" before Milvus Operator. (Line 354, 358, 366, etc)

Reply:
[b9a7150](https://github.com/milvus-io/milvus-docs/pull/1624/commits/b9a715009d5bbcd22dc38fd5c98bcb3e4adb606d)